app/views/judgment/judgmentFileChecksProgress.scala.html

```diff
@@ -0,0 +1,47 @@
+@import java.util.UUID
+@import viewsapi.FrontEndInfo
+@import views.html.partials.{frontEndInputs, progressIndicator}
+@(consignmentId: UUID, fileChecks: FileChecksProgress, frontEndInfo: FrontEndInfo)(implicit messages: Messages, request: RequestHeader)
+
+@main("Checking judgment") {
+    @defining(play.core.PlayVersion.current) { version =>
+    <div class="govuk-grid-row file-check-progress">
+        <div class="govuk-grid-column-two-thirds">
+
+            @progressIndicator(Messages("checkingRecords.progress"))
+            @frontEndInputs(frontEndInfo)
+            <h1 class="govuk-heading-l">Checking court judgment record</h1>
+            <input id="consignmentId" type="hidden" value="@consignmentId">
+            <p class="govuk-body">Please wait while your court judgment is being checked. This may take a few minutes.</p>
+            <p class="govuk-body">The following checks are now being performed:</p>
```

Comment:
I think we can remove the list of checks from the page. I seem to remember Kirsten saying this was what we're going to do for now.

Reply:
The design on the ticket has them but I remember now saying we'd get rid of them. I've taken them out.